airflow/models/dag.py

```diff
@@ -88,6 +89,23 @@ def get_last_dagrun(dag_id, session, include_externally_triggered=False):
     return query.first()
 
 
+def replace_outdated_dag_actions(access_control):
```

Comment:
I think make this an `@staticmethod` on dag, and then also call it `_upgrade_outdated_dag_access_control`